meta-lmp-bsp/dynamic-layers/meta-ti-bsp/recipes-ti/secdev/ti-k3-secdev_git.bb

```diff
@@ -4,7 +4,9 @@ SECTION = "devel"
 LICENSE = "BSD-3-Clause"
 LIC_FILES_CHKSUM = "file://manifest/k3-secdev-0.2-manifest.html;md5=f632a78870cc64550078d7b3cbac0892"
 
-require recipes-ti/includes/ti-paths.inc
+# set a default value for TI_K3_SECDEV_INSTALL_DIR_RECIPE
+export TI_K3_SECDEV_INSTALL_DIR_RECIPE = "${datadir}/ti/ti-k3-secdev"
+include recipes-ti/includes/ti-paths.inc
```

Comment:
With the export line we don't need this include anymore, but fine to keep if you prefer.

Reply:
right, this is because the patch is based on the upstream version. I think we can go with this and hope this PR can be reverted as soon as it is integrated upstream.